vignettes/contribution_model.Rmd

```diff
@@ -45,22 +45,55 @@ Our contribution model is still in a testing phase so we advise you to contact u
 knitr::include_graphics("./admiral_contribution_model.png")
 ```
 
-Type 1 Contribution with Code: 
-
- * First, the user creates an issue or comments on an existing issue to notify that they’d like to contribute code.
- 
- * Follow our [development
- process](https://pharmaverse.github.io/admiraldev/devel/articles/development_process.html)
- step-by-step guide.
- 
- * We advise to contact an `{admiral}` core development team directly via [Slack](https://app.slack.com/client/T028PB489D3/C02M8KN8269) before submitting code for complex functionality.
-
-
-Type 2 Contribution without Code: 
+Type 1 Contribution without Code: 
 
  * User creates an issue and ideally contacts an `{admiral}` team member via [Slack](https://app.slack.com/client/T028PB489D3/C02M8KN8269).
  
  * The `{admiral}` core development team will contact the issue creator as soon as possible to discuss further details.
+ 
+Type 2 Contribution with Code: 
 
-
-
+ * User creates an issue or comments on an existing issue to notify that they’d like to contribute code.
+ 
+ * We advise to contact the `{admiral}` core development team directly via [Slack](https://app.slack.com/client/T028PB489D3/C02M8KN8269) before submitting code for complex functionality.
+ 
+  * Once an issue is assigned to you by admiral core development team, follow our step-by-step guide in the development process section below, or checkout the [admiral contributor onboarding Youtube video](https://www.youtube.com/watch?v=YcDxBmlpykw&t=6s).
+
+
+## Development Process
+If you decide to contribute with code and you’re ready to make your first code 
+contribution, this development process step-by-step guide will help tie all the 
+other detailed vignettes together to give you the simplest experience of helping 
+to grow and enhance our codebase.
+
+1. Create a new feature branch from the development branch `main` following the naming convention and pull the latest changes - as detailed on the [github usage](git_usage.html#working-with-feature-branches-1) guide.
+1. Familiarize yourself with the `{admiral}` [programming
+strategy](programming_strategy.html), and then make the required code updates.
+1. Before making a pull request, check the [Pull Request Review Guidance](pr_review_guidance.html) & the following checklist of common things developers miss:
+    a. Is all your code formatted according to the [tidyverse](https://style.tidyverse.org/) style guide?
```

Comment:
Could we have fun with the itemized checklist?  Like emojiss or even a tasklist??  Just feels so boring but we are so fun!!

e.g. 
- [ ] 

:question:
:notes:
:star: